inc/Styles/Component.php

```diff
@@ -291,18 +291,6 @@ protected function get_css_files() : array {
 				'file'             => 'content.min.css',
 				'preload_callback' => '__return_true',
 			),
-			'wp-rig-sidebar'    => array(
-				'file'             => 'sidebar.min.css',
-				'preload_callback' => function() {
-					return wp_rig()->is_primary_sidebar_active();
-				},
-			),
-			'wp-rig-widgets'    => array(
-				'file'             => 'widgets.min.css',
-				'preload_callback' => function() {
-					return wp_rig()->is_primary_sidebar_active();
-				},
-			),
```

Comment:
Could we possibly just keep that bit and add the `method_exists()` checks inside the `preload_callback`? Or is there something I'm missing that wouldn't work then?

Reply:
If we only check inside the `preload_callback`, the styles will be enqueued when peloading styles is disabled. We don't need these files when there's no Sidebar component.